lib/provider.coffee

```diff
@@ -94,13 +100,19 @@ module.exports =
       hasScope(previousScopesArray, 'entity.other.attribute-name.id') or
       hasScope(previousScopesArray, 'entity.other.attribute-name.parent-selector.css') or
       hasScope(previousScopesArray, 'entity.name.tag.reference.scss') or
-      hasScope(previousScopesArray, 'entity.name.tag.scss')
+      hasScope(previousScopesArray, 'entity.name.tag.scss') or
+      hasScope(previousScopesArray, 'entity.name.tag.reference.postcss') or
+      hasScope(previousScopesArray, 'entity.name.tag.postcss')
 
     isAtBeginScopePunctuation = hasScope(scopes, 'punctuation.section.property-list.begin.bracket.curly.css') or
       hasScope(scopes, 'punctuation.section.property-list.begin.bracket.curly.scss') or
+      hasScope(scopes, 'punctuation.section.property-list.begin.bracket.curly.postcss') or
+      hasScope(scopes, 'punctuation.section.property-list.begin.postcss') or
```

Comment:
Honestly I don't remember, I developer it long time ago, but if I did so, surely it's there for a reason: i'll check and figure it out. In your opinion it's not needed?